trillian/ctfe/instance.go

```diff
@@ -35,13 +35,9 @@ import (
 // LogConfigFromFile creates a slice of LogConfig options from the given
 // filename, which should contain text-protobuf encoded configuration data.
 func LogConfigFromFile(filename string) ([]*configpb.LogConfig, error) {
-	if len(filename) == 0 {
-		return nil, errors.New("log config filename empty")
-	}
-
-	cfgText, err := ioutil.ReadFile(filename)
+	cfgText, err := readConfig(filename)
 	if err != nil {
-		return nil, fmt.Errorf("failed to read log config: %v", err)
+		return nil, err
```

Comment:
As mentioned below, maybe restore the `ioutil.ReadFile` / `fmt.Errorf` here but drop the `len(filename)==0` check.

Reply:
Done.